lib/compile.ml

```diff
@@ -26,9 +27,16 @@ let compile_fragment all_infos info =
 
   [ Format.asprintf "%a : %a %s" Fpath.pp odoc_path Fpath.pp (Inputs.input_file info) deps_str;
     Format.asprintf "\t@odoc compile --package %s $< %s -o %a" info.package include_str Fpath.pp odoc_path;
-    Format.asprintf "compile : %a" Fpath.pp odoc_path;
+    Format.asprintf "compile-%s : %a" info.package Fpath.pp odoc_path;
```

Comment:
We should mark this target as phony, or make it a real (but hidden, usually) stamp file.

Reply:
That's done in the function below, where we have the list of those targets.